src/installation/guides/arm-devices/index.md

```diff
@@ -0,0 +1,137 @@
+# ARM Devices
+
+Void Linux provides packages and images for several ARM devices. Installing Void
+on such devices can be done in several ways:
+
+- [Pre-built images](#pre-built-images): images that can be flashed directly
+   onto an SD card or other storage medium, but give you a limited partition
+   layout and require manual expansion if you wish to increase the partitions;
+- [Tarball installation](#tarball-installation): PLATFORMFS and ROOTFS tarballs
+   that can be extracted to a previously prepared partition scheme;
+- [Chroot installation](#chroot-installation): follows most of the steps
+   outlined in [the chroot guide](../chroot.md).
+
+Since most of the commands on this guide will be run on external storage, it is
+important to run [sync(1)](https://man.voidlinux.org/sync.1) before removing the
+device.
+
+Further [configuration steps](#configuration) that are mostly specific to such
+devices are also outlined.
+
+## Installation
+
+If you are installing Void Linux on one of the ARM devices covered in the
+"[Supported platforms](./platforms.md)" page, make sure to read its section
+thoroughly.
+
+### Pre-built images
+
+After [downloading and verifying](../../index.md#downloading-installation-media)
+an image, it can be flashed onto an SD card in `/dev/mmcblk0`, for example, with
+the [dd(1)](https://man.voidlinux.org/dd.1) command:
+
+```
+# dd if=<image>.img of=/dev/mmcblk0 bs=4M status=progress
```

Comment:
1. mention `sync`?
2. in fact you can just `cat` or `pv` the image to the blockdevice

Reply:
I have this in the beginning:

```
Since most of the commands on this guide will be run on external storage, it is
important to run [sync(1)](https://man.voidlinux.org/sync.1) before removing the
device.
```

I always do `dd` for images. For `cat` it's just `cat <image> > /dev/mmcblk0`, right? I can add that.

Reply:
Added cat.

Reply:
suggestion: `with the cat, pv or  [dd(1)](https://man.voidlinux.org/dd.1) command:`
For sure we don't want to list all possible code snippets.
I'm open to not mention `cat` at all too